### ISSUE TITLE
attr(column, 'label')

### ISSUE THREAD

**Issue by vincentarelbundock** [nset-ornl/wbstats#13]
If you assign an indicator's description to its column's `label` attribute, RStudio will display it in the data browser.

For example,

```r
attr(dat[['NY.GDP.PCAP.KD']], 'label') <- 'GDP per capita (constant 2010 US$)'
```

Gives something like this:

<img width="890" alt="screen shot 2019-02-03 at 8 21 37 pm" src="https://user-images.githubusercontent.com/987057/52185838-7aa8df80-27f1-11e9-9f84-a311b5e417bf.png">

**Comment by jpiburn** [comment]
Another good idea. Thanks! added to the list

**Comment by jpiburn** [comment]
Implemented as of version `1.0`. Thanks again for the suggestion
